src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -1035,11 +1036,11 @@ create_from_json(#wm_reqdata{} = Req,
     %% a 500 and client can retry. If we succeed and the db call fails or conflicts, we can
     %% safely send a delete to solr since this is a new object with a unique ID unknown to
     %% the world.
-    ok = oc_chef_object_db:add_to_solr(ObjectRec, ObjectEjson),
+    ok = oc_chef_object_db:add_to_solr(ObjectRec, ObjectEjson, ReqId),
```

Comment:
Looks like it was added so that stats_hero could be used to time solr requests:

https://github.com/chef/chef-server/pull/520/files#diff-164d5c5b7e47072b0967d26bee658451R57